manifests/examples/virtualbox/jessie-vagrant.yml

```diff
@@ -4,7 +4,7 @@ provider:
   name: virtualbox
   guest_additions: /usr/share/virtualbox/VBoxGuestAdditions.iso
 bootstrapper:
-  workspace: /target
+  workspace: /target/jessie
```

Comment:
You don't have to create a subfolder in the workspace directory. This is [already generated in a random way][1].

[1]: https://github.com/andsens/bootstrap-vz/blob/72699d09c8d18fccb6f434023f33c92ca4ad94ca/bootstrapvz/base/bootstrapinfo.py#L19